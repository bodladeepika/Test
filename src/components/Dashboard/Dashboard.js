import React,{Fragment} from 'react';
import StickyHeader from '../Header/StickyHeader'
import Sidebar from '../Sidebar/Sidebar'
import faker from 'faker'
import './dashboard.scss'

export default () => {
  const [sideBarExpanded, setSideBarExpanded] = React.useState(true);
  const width = sideBarExpanded ? '15%':'3.5%'
  const mainContentStyle = {
    marginLeft: sideBarExpanded ? "15%" : "3.5%", 
    transition: "margin 0.2s ease"
  };

  return ( <Fragment>
    <StickyHeader/>
    <div className="main-content">
      
        <Sidebar width={width} setSideBarExpanded={setSideBarExpanded} sideBarExpanded={sideBarExpanded} />
        
       <div className="container-fluid"> 
            
            <div className="dashboard body" style={mainContentStyle}>
            <p>{faker.lorem.paragraphs()}</p>
            <p>{faker.lorem.paragraphs()}</p>
            <p>{faker.lorem.paragraphs()}</p>
            <p>{faker.lorem.paragraphs()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            <p>{faker.lorem.paragraph()}</p>
            </div>

        </div>

     </div>
   
   
  
    </Fragment>

  ) 
}