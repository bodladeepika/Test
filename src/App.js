import React from 'react';
import Dashboard from './components/Dashboard/Dashboard';

export default () => {
    return (<div> 
    
    <Dashboard/>

    </div>) 
}
