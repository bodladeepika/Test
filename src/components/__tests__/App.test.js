import React from 'react';
import Dashboard from '../Dashboard/Dashboard'
import {shallow} from 'enzyme'
import App from '../../App';
import toJson from 'enzyme-to-json';
import '@testing-library/jest-dom/extend-expect';


let wrapped;
beforeEach ( () => {
    wrapped = shallow(<App/>);
})

afterEach ( () => {
    wrapped.unmount()
})


it('has another component' , () => {

expect(wrapped.containsMatchingElement(<Dashboard />)).toEqual(true);
})

it('has correclty rendered',()=>{
  expect(toJson(wrapped)).toMatchSnapshot()
})
