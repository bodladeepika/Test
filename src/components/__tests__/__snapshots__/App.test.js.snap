// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has correclty rendered 1`] = `
<div>
  <_default />
</div>
`;
